src/test/java/io/nats/client/impl/KeyValueTests.java

```diff
@@ -383,6 +383,11 @@ public void testKeys() throws Exception {
             for (int x = 2; x <= 10; x += 2) {
                 assertTrue(keys.contains("k" + x));
             }
+
+            String keyWithDot = "part1.part2";
+            kv.put(keyWithDot, "key has dot");
+            KeyValueEntry kve = kv.get(keyWithDot);
```

Comment:
@scottf 
Thank you for the fast reply!
This test works already before the change in NatsKeyValueUtil. A additional test for the keys in kv.keys() would be better.
Mabye:
```
List<String> keys = kv.keys();
assertEquals(1, keys.size());
assertEquals(keyWithDot, keys.get(0));
```